### ISSUE TITLE
Hampran

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/tfc-nusantara-epigraphy#70]
@danbalogh — could you take look at https://dharmalekha.info/texts/INSIDENKHampran? As a reminder, the estampage was shown in https://github.com/erc-dharma/project-documentation/issues/193. Is it necessary to apply the textpart structure also in translation? Do you approve of my use of `<milestone>`s in textpart 2?

@aditiagunawan @wayanjarrah @zakariyaaminullah @tyassanti @ekobastiawan : could you take a look at the estampage shown in https://github.com/erc-dharma/project-documentation/issues/193 and give me your opinion on whether the aksaras in B5 use the same script as those in zone C? And can anyone get meaningful words out of B5? (I think it would be worth trying to read it as if it is Merapi-Merbabu script, which some of you know better than I do.) As long as I can't read that line, I don't know whether I have correctly assigned kaw-Latn to textpart 2. Has this part of this inscription figured in any of the publications by Abimarda dll. on the "Merapi-Merbabu" dating system with these kinds of diagrams and circles?

@wayanjarrah, @ekobastiawan @tyassanti : you will be interested to look at the details of my encoding (press the Source view button). I have used some elements from our TEI toolbox that we don't often use. You will also be interested to see how the opening of this line of bibliography is generated with a method we have recently introduced defining a "Shorthand" in the Zotero Extra field (see gdoc of ZG, §4.15).

<img width="1179" alt="Capture d’écran 2024-05-18 à 18 59 30" src="https://github.com/erc-dharma/tfc-nusantara-epigraphy/assets/835603/95bce77d-c957-4f0b-bf66-5be3d2854d4f">

Thanks!

Arlo

**Comment by danbalogh** [comment]
Yes, it is necessary to apply the textpart structure in translation. EGD §9.2.5, "if your edition includes boxlike partitions (§3.4), the textpart divisions of your edition must be mandatorily replicated in your translation (with the same attributes and, if applicable, <head> elements)".

I'm OK with your use of milestones in textpart 2. I've looked up our earlier discussions on this (in issue #193 and my email reply to you on 1 April 2020) and see that your original inclination was to use separate textparts for each of the little bits but you felt that this was discouraged by the EGD. Back then, I had said that in my opinion the EGD does not forbid doing so, and it seems that I had also made a couple of changes in the EGD text to reduce that discouraging. So I can only repeat that now: there is no strict rule for such unusual cases, and the choice is yours. I'm equally happy with milestones in textpart 2 or multiple textparts instead of textpart 2. If I was doing the encoding of this inscription, I would probably do the latter, but my preference is not strong or definite enough to suggest that you now revise your encoding. If you are happy with it, let it stay.

**Comment by arlogriffiths** [comment]
thanks Dan. I await responses from the others.

**Comment by wayanjarrah** [comment]
@arlogriffiths I feel that the aksaras used in zone C are the same as those in B5. Sorry I can't offer anything more helpful for B5.

Thank you for showing us the Shorthand tactic for NBG. Presumably we will want to apply this across the board for this periodical which is very often mentioned in our inscription's bibliographies. I'm happy to get started on adding the field for the NBG list for now.
